### PR TITLE
Regenerated for Azure.Storage.Files.Shares

### PR DESCRIPTION
**Pull request by amnguye**
Let me know if I need to make updates to swagger.

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs [hunk 1]
```diff
@@ -898,6 +898,7 @@ public static partial class BlobsModelFactory
         public static Azure.Storage.Blobs.Models.BlockInfo BlockInfo(byte[] contentHash, byte[] contentCrc64, string encryptionKeySha256) { throw null; }
         public static Azure.Storage.Blobs.Models.BlockInfo BlockInfo(byte[] contentHash, byte[] contentCrc64, string encryptionKeySha256, string encryptionScope) { throw null; }
         public static Azure.Storage.Blobs.Models.BlockList BlockList(System.Collections.Generic.IEnumerable<Azure.Storage.Blobs.Models.BlobBlock> committedBlocks = null, System.Collections.Generic.IEnumerable<Azure.Storage.Blobs.Models.BlobBlock> uncommittedBlocks = null) { throw null; }
+        public static Azure.Storage.Blobs.Models.ContainerSubmitBatchResult ContainerSubmitBatchResult(string contentType, System.IO.Stream body) { throw null; }
```
Comment (seanmcc-msft) [inline sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs:901]: Can we hold off merging this?  The swagger readme transforms to hide stuff like this are in the stg76base branch, which I will merge next week.  https://github.com/Azure/azure-sdk-for-net/pull/18068

File: sdk/storage/Azure.Storage.Files.Shares/src/Generated/FileRestClient.cs [hunk 2]
```diff
@@ -11519,10 +11519,10 @@ internal ShareProtocolSettings(bool skipInitialization)
         /// Serialize a ShareProtocolSettings instance as XML.
         /// </summary>
         /// <param name="value">The ShareProtocolSettings instance to serialize.</param>
-        /// <param name="name">An optional name to use for the root element instead of "ShareProtocolSettings".</param>
+        /// <param name="name">An optional name to use for the root element instead of "ProtocolSettings".</param>
         /// <param name="ns">An optional namespace to use for the root element instead of "".</param>
         /// <returns>The serialized XML element.</returns>
-        internal static System.Xml.Linq.XElement ToXml(Azure.Storage.Files.Shares.Models.ShareProtocolSettings value, string name = "ShareProtocolSettings", string ns = "")
+        internal static System.Xml.Linq.XElement ToXml(Azure.Storage.Files.Shares.Models.ShareProtocolSettings value, string name = "ProtocolSettings", string ns = "")
```
Comment (kasobol-msft) [inline sdk/storage/Azure.Storage.Files.Shares/src/Generated/FileRestClient.cs:11525]: we should add test coverage for this (and the other change).

Comment (amnguye) [reply sdk/storage/Azure.Storage.Files.Shares/src/Generated/FileRestClient.cs:11525]: I should add another transform to switch this back actually

Comment (amnguye) [reply sdk/storage/Azure.Storage.Files.Shares/src/Generated/FileRestClient.cs:11525]: Nevermind, I'm wrong it should be this.

**Review by kasobol-msft — APPROVED** [review]

**Review by seanmcc-msft — APPROVED** [review]

**Pull request merged**